packages/astro/package.json

```diff
@@ -122,6 +130,7 @@
     "cookie": "^0.5.0",
     "debug": "^4.3.4",
     "deepmerge-ts": "^4.2.2",
+    "devalue": "^4.2.0",
```

Comment:
`devalue` was added to stringify Zod schema results for injection in Vite virtual modules. Ideally, we would extract our props serialization logic to something generic that could replace `devalue` here. That refactor seemed a _bit_ intense for an already large PR, so I'd like to tackle as a follow-up.